app/ide-desktop/lib/dashboard/src/layouts/InfoMenu.tsx

```diff
@@ -0,0 +1,81 @@
+/** @file A menu containing info about the app. */
+import * as React from 'react'
+
+import LogoIcon from 'enso-assets/enso_logo.svg'
+import * as common from 'enso-common'
+
+import * as modalProvider from '#/providers/ModalProvider'
+import * as textProvider from '#/providers/TextProvider'
+
+import * as aria from '#/components/aria'
+import MenuEntry from '#/components/MenuEntry'
+import Modal from '#/components/Modal'
+import FocusArea from '#/components/styled/FocusArea'
+
+import AboutModal from '#/modals/AboutModal'
+
+// ================
+// === InfoMenu ===
+// ================
+
+/** Props for an {@link InfoMenu}. */
+export interface InfoMenuProps {
+  readonly hidden?: boolean
+}
+
+/** A menu containing info about the app. */
+export default function InfoMenu(props: InfoMenuProps) {
+  const { hidden = false } = props
+  const { setModal } = modalProvider.useSetModal()
+  const { getText } = textProvider.useText()
+  const [initialized, setInitialized] = React.useState(false)
+
+  React.useEffect(() => {
+    requestAnimationFrame(() => {
+      setInitialized(true)
+    })
+  }, [])
+
+  return (
+    <Modal hidden={hidden} className="absolute size-full overflow-hidden bg-dim">
+      <div
+        // The name comes from a third-party API and cannot be changed.
+        // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
We definetely should allow this in eslint. Adding eslint-disable and reason each time is annoying

Reply:
annoying, but correct.
but we *can* definitely add specific exceptions